source/includes/api/_areas_v2.md

```diff
@@ -45,6 +61,39 @@ curl -X GET https://api.resourcewatch.org/v2/area
 }
 ```
 
+The `v2/areas` endpoint returns all the areas of interest associated with the user who made the request. 
+
+**This endpoint requires authentication, and it will return 401 Unauthorized if none is provided.**
```

Comment:
Replace this type of notice with an "Errors for ..." section, like we've added to recent documentation refactors

Reply:
Also add other potential errors users may run into when using each respective endpoint

Reply:
Done and done :+1: